GCDWebUploader/GCDWebUploader.m

```diff
@@ -296,7 +296,13 @@ @implementation GCDWebUploader
 
 - (instancetype)initWithUploadDirectory:(NSString*)path {
   if ((self = [super init])) {
-    NSBundle* siteBundle = [NSBundle bundleWithPath:[[NSBundle mainBundle] pathForResource:@"GCDWebUploader" ofType:@"bundle"]];
+    NSString *bundlePath = [[NSBundle mainBundle] pathForResource:@"GCDWebUploader"
+                                                           ofType:@"bundle"];
+    if (!bundlePath) {
```

Comment:
I kept it because it leave the door open for changing `GCDWebUploader.bundle` so it search first on the main bundle (so a developer can change the bundle) and then on the framework's resources for the default one